data/BookingsDataExtracts/BookingsDataExtract.cs

```diff
@@ -0,0 +1,62 @@
+using BookingsDataExtracts.Documents;
+using Parquet;
+using Parquet.Schema;
+
+namespace BookingsDataExtracts;
+
+public class BookingDataExtract(
+    CosmosStore<BookingDocument> bookingsStore, 
+    CosmosStore<SiteDocument> sitesStore,
+    TimeProvider timeProvider)
+{
+    public async Task RunAsync(FileInfo outputFile)
+    {
+        Console.WriteLine("Loading bookings");
+        
+        var allBookings = await bookingsStore.RunQueryAsync(b => b.DocumentType == "booking" && b.StatusUpdated > timeProvider.GetUtcNow().Date.AddDays(-1) && b.StatusUpdated < timeProvider.GetUtcNow().Date, b => b);
+        var bookings = allBookings.Where(b => b.Status != AppointmentStatus.Provisional).ToList();
```

Comment:
I noticed in the `BookingDataConverter` we throw an out of range exception if booking isn't Cancelled or Booked. My open PR adds in an additional `Orphaned` status, so there's a chance that exception may get thrown if that goes in. 
I'm not suggesting you make any chances until/unless mine gets merged since my PR could always change during review, and also the table in the ticket says this must be Booked | Cancelled. I just wanted to flag it